engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version-module-streams.html

```diff
@@ -0,0 +1,27 @@
+<div data-extend-template="layouts/partials/table.html">
+  <div data-block="search-filter">
+    <select class="form-control" ng-model="repository" ng-options="repository.name for (id, repository) in repositories"></select>
+  </div>
+
+  <table data-block="table" class="table table-striped table-bordered">
+    <thead>
+      <tr bst-table-head>
+        <th bst-table-column translate>Name</th>
+        <th bst-table-column translate>Stream</th>
+        <th bst-table-column translate>Version</th>
+        <th bst-table-column translate>Context</th>
+        <th bst-table-column translate>Arch</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <tr bst-table-row ng-repeat="module in table.rows">
+        <td bst-table-cell>{{ module.name }}</td>
+        <td bst-table-cell>{{ module.stream }}</td>
+        <td bst-table-cell>{{ module.version }}</td>
+        <td bst-table-cell>{{ module.context }}</td>
+        <td bst-table-cell>{{ module.arch }}</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
```

Comment:
If this is the same as the other PR, you can make this a shared template/partial

Reply:
good call